// animal-crossing/lib/index.js
"use strict";
// ---
// TYPES
// ---
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.npcs = exports.villagers = exports.translations = exports.recipes = exports.seasonsAndEvents = exports.reactions = exports.items = exports.creatures = exports.construction = exports.achievements = void 0;
// ---
// SOURCE FILES
// ---
const Achievements_json_1 = __importDefault(require("./data/Achievements.json"));
const Construction_json_1 = __importDefault(require("./data/Construction.json"));
const Creatures_json_1 = __importDefault(require("./data/Creatures.json"));
const Items_json_1 = __importDefault(require("./data/Items.json"));
const Reactions_json_1 = __importDefault(require("./data/Reactions.json"));
const Recipes_json_1 = __importDefault(require("./data/Recipes.json"));
const SeasonsAndEvents_json_1 = __importDefault(require("./data/SeasonsAndEvents.json"));
const Translations_json_1 = __importDefault(require("./data/Translations.json"));
const Villagers_json_1 = __importDefault(require("./data/Villagers.json"));
const NPCs_json_1 = __importDefault(require("./data/NPCs.json"));
// ---
// EXPORTS
// ---
exports.achievements = Achievements_json_1.default;
exports.construction = Construction_json_1.default;
exports.creatures = Creatures_json_1.default;
exports.items = Items_json_1.default;
exports.reactions = Reactions_json_1.default;
exports.seasonsAndEvents = SeasonsAndEvents_json_1.default;
exports.recipes = Recipes_json_1.default;
exports.translations = Translations_json_1.default;
exports.villagers = Villagers_json_1.default;
exports.npcs = NPCs_json_1.default;
