// animal-crossing/lib/types/Construction.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.Version = exports.TranslationsSourceSheet = exports.ConstructionSourceSheet = exports.Source = exports.Category = void 0;
var Category;
(function (Category) {
    Category["Bridge"] = "Bridge";
    Category["Door"] = "Door";
    Category["Incline"] = "Incline";
    Category["Mailbox"] = "Mailbox";
    Category["Roofing"] = "Roofing";
    Category["Siding"] = "Siding";
})(Category = exports.Category || (exports.Category = {}));
var Source;
(function (Source) {
    Source["InitialHouse"] = "Initial House";
    Source["ResidentServicesUpgrade"] = "Resident Services Upgrade";
    Source["Tent"] = "Tent";
    Source["The3RDHouseUpgradeLeftRoom"] = "3rd House Upgrade (Left Room)";
    Source["The4ThHouseUpgradeRightRoom"] = "4th House Upgrade (Right Room)";
    Source["The5ThHouseUpgrade2NdFloor"] = "5th House Upgrade (2nd Floor)";
})(Source = exports.Source || (exports.Source = {}));
var ConstructionSourceSheet;
(function (ConstructionSourceSheet) {
    ConstructionSourceSheet["Construction"] = "Construction";
})(ConstructionSourceSheet = exports.ConstructionSourceSheet || (exports.ConstructionSourceSheet = {}));
var TranslationsSourceSheet;
(function (TranslationsSourceSheet) {
    TranslationsSourceSheet["BridgeInclines"] = "Bridge & Inclines";
    TranslationsSourceSheet["HouseDoor"] = "House Door";
    TranslationsSourceSheet["HouseMailbox"] = "House Mailbox";
    TranslationsSourceSheet["HouseRoof"] = "House Roof";
    TranslationsSourceSheet["HouseWall"] = "House Wall";
})(TranslationsSourceSheet = exports.TranslationsSourceSheet || (exports.TranslationsSourceSheet = {}));
var Version;
(function (Version) {
    Version["The100"] = "1.0.0";
})(Version = exports.Version || (exports.Version = {}));
