// animal-crossing/lib/types/Creature.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.Weather = exports.Vision = exports.Version = exports.TranslationsSourceSheet = exports.CreatureSourceSheet = exports.Size = exports.Shadow = exports.MovementSpeed = exports.LightingType = exports.HhaCategory = exports.Color = exports.CatchDifficulty = void 0;
var CatchDifficulty;
(function (CatchDifficulty) {
    CatchDifficulty["Easy"] = "Easy";
    CatchDifficulty["Hard"] = "Hard";
    CatchDifficulty["Medium"] = "Medium";
    CatchDifficulty["VeryEasy"] = "Very Easy";
    CatchDifficulty["VeryHard"] = "Very Hard";
})(CatchDifficulty = exports.CatchDifficulty || (exports.CatchDifficulty = {}));
var Color;
(function (Color) {
    Color["Aqua"] = "Aqua";
    Color["Beige"] = "Beige";
    Color["Black"] = "Black";
    Color["Blue"] = "Blue";
    Color["Brown"] = "Brown";
    Color["Gray"] = "Gray";
    Color["Green"] = "Green";
    Color["Orange"] = "Orange";
    Color["Pink"] = "Pink";
    Color["Purple"] = "Purple";
    Color["Red"] = "Red";
    Color["White"] = "White";
    Color["Yellow"] = "Yellow";
})(Color = exports.Color || (exports.Color = {}));
var HhaCategory;
(function (HhaCategory) {
    HhaCategory["Pet"] = "Pet";
})(HhaCategory = exports.HhaCategory || (exports.HhaCategory = {}));
var LightingType;
(function (LightingType) {
    LightingType["Emission"] = "Emission";
    LightingType["Fluorescent"] = "Fluorescent";
})(LightingType = exports.LightingType || (exports.LightingType = {}));
var MovementSpeed;
(function (MovementSpeed) {
    MovementSpeed["Fast"] = "Fast";
    MovementSpeed["Medium"] = "Medium";
    MovementSpeed["Slow"] = "Slow";
    MovementSpeed["Stationary"] = "Stationary";
    MovementSpeed["VeryFast"] = "Very fast";
    MovementSpeed["VerySlow"] = "Very slow";
})(MovementSpeed = exports.MovementSpeed || (exports.MovementSpeed = {}));
var Shadow;
(function (Shadow) {
    Shadow["Large"] = "Large";
    Shadow["Long"] = "Long";
    Shadow["Medium"] = "Medium";
    Shadow["Small"] = "Small";
    Shadow["XLarge"] = "X-Large";
    Shadow["XLargeWFin"] = "X-Large w/Fin";
    Shadow["XSmall"] = "X-Small";
    Shadow["XXLarge"] = "XX-Large";
})(Shadow = exports.Shadow || (exports.Shadow = {}));
var Size;
(function (Size) {
    Size["The1X1"] = "1x1";
    Size["The2X1"] = "2x1";
    Size["The2X2"] = "2x2";
    Size["The3X2"] = "3x2";
})(Size = exports.Size || (exports.Size = {}));
var CreatureSourceSheet;
(function (CreatureSourceSheet) {
    CreatureSourceSheet["Fish"] = "Fish";
    CreatureSourceSheet["Insects"] = "Insects";
    CreatureSourceSheet["SeaCreatures"] = "Sea Creatures";
})(CreatureSourceSheet = exports.CreatureSourceSheet || (exports.CreatureSourceSheet = {}));
var TranslationsSourceSheet;
(function (TranslationsSourceSheet) {
    TranslationsSourceSheet["Bugs"] = "Bugs";
    TranslationsSourceSheet["Fish"] = "Fish";
    TranslationsSourceSheet["SeaCreatures"] = "Sea Creatures";
})(TranslationsSourceSheet = exports.TranslationsSourceSheet || (exports.TranslationsSourceSheet = {}));
var Version;
(function (Version) {
    Version["The100"] = "1.0.0";
    Version["The130"] = "1.3.0";
})(Version = exports.Version || (exports.Version = {}));
var Vision;
(function (Vision) {
    Vision["Medium"] = "Medium";
    Vision["Narrow"] = "Narrow";
    Vision["VeryNarrow"] = "Very Narrow";
    Vision["VeryWide"] = "Very Wide";
    Vision["Wide"] = "Wide";
})(Vision = exports.Vision || (exports.Vision = {}));
var Weather;
(function (Weather) {
    Weather["AnyExceptRain"] = "Any except rain";
    Weather["AnyWeather"] = "Any weather";
    Weather["RainOnly"] = "Rain only";
})(Weather = exports.Weather || (exports.Weather = {}));
