// animal-crossing/lib/types/Item.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.WindowType = exports.WindowColor = exports.VfxType = exports.VariantTranslationsSourceSheet = exports.KitType = exports.VariantID = exports.Spanish = exports.ComfySourceSheet = exports.Russian = exports.Korean = exports.Japanese = exports.Italian = exports.ID = exports.German = exports.French = exports.DutchEnum = exports.ChineseTraditional = exports.Chinese = exports.Theme = exports.Style = exports.SpeakerType = exports.SizeCategory = exports.Size = exports.SecondaryShape = exports.RecipeSourceSheet = exports.Version = exports.SeriesTranslationsSourceSheet = exports.CardColor = exports.PrimaryShape = exports.PaneType = exports.Museum = exports.SeasonalAvailability = exports.LightingType = exports.InteractEnum = exports.HhaCategory = exports.Gender = exports.ExchangeCurrency = exports.CurtainType = exports.Concept = exports.Color = exports.CeilingType = exports.Category = exports.Catalog = void 0;
var Catalog;
(function (Catalog) {
    Catalog["ForSale"] = "For sale";
    Catalog["NotForSale"] = "Not for sale";
})(Catalog = exports.Catalog || (exports.Catalog = {}));
var Category;
(function (Category) {
    Category["Accessories"] = "Accessories";
    Category["Art"] = "Art";
    Category["Bags"] = "Bags";
    Category["Bottoms"] = "Bottoms";
    Category["ClothingOther"] = "Clothing Other";
    Category["DressUp"] = "Dress-Up";
    Category["Equipment"] = "Equipment";
    Category["Fencing"] = "Fencing";
    Category["Floors"] = "Floors";
    Category["Fossils"] = "Fossils";
    Category["Headwear"] = "Headwear";
    Category["Housewares"] = "Housewares";
    Category["MessageCards"] = "Message Cards";
    Category["Miscellaneous"] = "Miscellaneous";
    Category["Music"] = "Music";
    Category["Other"] = "Other";
    Category["Photos"] = "Photos";
    Category["Posters"] = "Posters";
    Category["Rugs"] = "Rugs";
    Category["Shoes"] = "Shoes";
    Category["Socks"] = "Socks";
    Category["Tools"] = "Tools";
    Category["Tops"] = "Tops";
    Category["Umbrellas"] = "Umbrellas";
    Category["WallMounted"] = "Wall-mounted";
    Category["Wallpaper"] = "Wallpaper";
})(Category = exports.Category || (exports.Category = {}));
var CeilingType;
(function (CeilingType) {
    CeilingType["Cloth"] = "Cloth";
    CeilingType["Gold"] = "Gold";
    CeilingType["Plain"] = "Plain";
    CeilingType["Stone"] = "Stone";
    CeilingType["Tile"] = "Tile";
    CeilingType["Wood"] = "Wood";
})(CeilingType = exports.CeilingType || (exports.CeilingType = {}));
var Color;
(function (Color) {
    Color["Aqua"] = "Aqua";
    Color["Beige"] = "Beige";
    Color["Black"] = "Black";
    Color["Blue"] = "Blue";
    Color["Brown"] = "Brown";
    Color["Colorful"] = "Colorful";
    Color["Gray"] = "Gray";
    Color["Green"] = "Green";
    Color["Orange"] = "Orange";
    Color["Pink"] = "Pink";
    Color["Purple"] = "Purple";
    Color["Red"] = "Red";
    Color["White"] = "White";
    Color["Yellow"] = "Yellow";
})(Color = exports.Color || (exports.Color = {}));
var Concept;
(function (Concept) {
    Concept["Bathroom"] = "bathroom";
    Concept["ChildSRoom"] = "child's room";
    Concept["Concert"] = "concert";
    Concept["Den"] = "den";
    Concept["Expensive"] = "expensive";
    Concept["Facility"] = "facility";
    Concept["Fancy"] = "fancy";
    Concept["Fitness"] = "fitness";
    Concept["FreezingCold"] = "freezing cold";
    Concept["Garden"] = "garden";
    Concept["Harmonious"] = "harmonious";
    Concept["Heritage"] = "heritage";
    Concept["Horror"] = "horror";
    Concept["Kitchen"] = "kitchen";
    Concept["LivingRoom"] = "living room";
    Concept["Music"] = "music";
    Concept["None"] = "none";
    Concept["Ocean"] = "ocean";
    Concept["Office"] = "office";
    Concept["Outdoors"] = "outdoors";
    Concept["Party"] = "party";
    Concept["School"] = "school";
    Concept["Shop"] = "shop";
    Concept["Space"] = "space";
    Concept["Workshop"] = "workshop";
})(Concept = exports.Concept || (exports.Concept = {}));
var CurtainType;
(function (CurtainType) {
    CurtainType["Curtains"] = "Curtains";
    CurtainType["RollerShades"] = "Roller Shades";
    CurtainType["SlattedBlinds"] = "Slatted Blinds";
})(CurtainType = exports.CurtainType || (exports.CurtainType = {}));
var ExchangeCurrency;
(function (ExchangeCurrency) {
    ExchangeCurrency["HeartCrystals"] = "Heart Crystals";
    ExchangeCurrency["NookMiles"] = "Nook Miles";
})(ExchangeCurrency = exports.ExchangeCurrency || (exports.ExchangeCurrency = {}));
var Gender;
(function (Gender) {
    Gender["Free"] = "Free";
    Gender["Manly"] = "Manly";
    Gender["Womanly"] = "Womanly";
})(Gender = exports.Gender || (exports.Gender = {}));
var HhaCategory;
(function (HhaCategory) {
    HhaCategory["AC"] = "AC";
    HhaCategory["Appliance"] = "Appliance";
    HhaCategory["Audio"] = "Audio";
    HhaCategory["Clock"] = "Clock";
    HhaCategory["Doll"] = "Doll";
    HhaCategory["Dresser"] = "Dresser";
    HhaCategory["Lighting"] = "Lighting";
    HhaCategory["MusicalInstrument"] = "MusicalInstrument";
    HhaCategory["Pet"] = "Pet";
    HhaCategory["Plant"] = "Plant";
    HhaCategory["SmallGoods"] = "SmallGoods";
    HhaCategory["Trash"] = "Trash";
    HhaCategory["Tv"] = "TV";
})(HhaCategory = exports.HhaCategory || (exports.HhaCategory = {}));
var InteractEnum;
(function (InteractEnum) {
    InteractEnum["Chair"] = "Chair";
    InteractEnum["Trash"] = "Trash";
    InteractEnum["Wardrobe"] = "Wardrobe";
    InteractEnum["Workbench"] = "Workbench";
})(InteractEnum = exports.InteractEnum || (exports.InteractEnum = {}));
var LightingType;
(function (LightingType) {
    LightingType["Candle"] = "Candle";
    LightingType["Emission"] = "Emission";
    LightingType["Fluorescent"] = "Fluorescent";
    LightingType["Monitor"] = "Monitor";
    LightingType["Spotlight"] = "Spotlight";
})(LightingType = exports.LightingType || (exports.LightingType = {}));
var SeasonalAvailability;
(function (SeasonalAvailability) {
    SeasonalAvailability["AllYear"] = "All Year";
    SeasonalAvailability["Autumn"] = "Autumn";
    SeasonalAvailability["Fall"] = "Fall";
    SeasonalAvailability["Spring"] = "Spring";
    SeasonalAvailability["Summer"] = "Summer";
    SeasonalAvailability["Winter"] = "Winter";
})(SeasonalAvailability = exports.SeasonalAvailability || (exports.SeasonalAvailability = {}));
var Museum;
(function (Museum) {
    Museum["Room1"] = "Room 1";
    Museum["Room2"] = "Room 2";
    Museum["Room3"] = "Room 3";
})(Museum = exports.Museum || (exports.Museum = {}));
var PaneType;
(function (PaneType) {
    PaneType["Glass"] = "Glass";
    PaneType["Screen"] = "Screen";
})(PaneType = exports.PaneType || (exports.PaneType = {}));
var PrimaryShape;
(function (PrimaryShape) {
    PrimaryShape["ALine"] = "A-line";
    PrimaryShape["ALong"] = "A-long";
    PrimaryShape["BLong"] = "B-long";
    PrimaryShape["Balloon"] = "Balloon";
    PrimaryShape["Box"] = "Box";
    PrimaryShape["Dress"] = "Dress";
    PrimaryShape["Kimono"] = "Kimono";
    PrimaryShape["Marinesuit"] = "Marinesuit";
    PrimaryShape["Overall"] = "Overall";
    PrimaryShape["Rib"] = "Rib";
    PrimaryShape["Robe"] = "Robe";
    PrimaryShape["Salopette"] = "Salopette";
})(PrimaryShape = exports.PrimaryShape || (exports.PrimaryShape = {}));
var CardColor;
(function (CardColor) {
    CardColor["Beige"] = "beige";
    CardColor["Blue"] = "blue";
    CardColor["Brick"] = "brick";
    CardColor["Brown"] = "brown";
    CardColor["Cream"] = "cream";
    CardColor["DarkGray"] = "dark gray";
    CardColor["Gold"] = "gold";
    CardColor["Green"] = "green";
    CardColor["LightGray"] = "light gray";
    CardColor["Orange"] = "orange";
    CardColor["Pink"] = "pink";
    CardColor["Purple"] = "purple";
    CardColor["Red"] = "red";
    CardColor["Silver"] = "silver";
    CardColor["White"] = "white";
    CardColor["Yellow"] = "yellow";
})(CardColor = exports.CardColor || (exports.CardColor = {}));
var SeriesTranslationsSourceSheet;
(function (SeriesTranslationsSourceSheet) {
    SeriesTranslationsSourceSheet["Accessories"] = "Accessories";
    SeriesTranslationsSourceSheet["Art"] = "Art";
    SeriesTranslationsSourceSheet["Bags"] = "Bags";
    SeriesTranslationsSourceSheet["Bottoms"] = "Bottoms";
    SeriesTranslationsSourceSheet["BugsModels"] = "Bugs Models";
    SeriesTranslationsSourceSheet["Caps"] = "Caps";
    SeriesTranslationsSourceSheet["Craft"] = "Craft";
    SeriesTranslationsSourceSheet["Doorplates"] = "Doorplates";
    SeriesTranslationsSourceSheet["Dresses"] = "Dresses";
    SeriesTranslationsSourceSheet["Etc"] = "ETC";
    SeriesTranslationsSourceSheet["EventItems"] = "Event Items";
    SeriesTranslationsSourceSheet["Fence"] = "Fence";
    SeriesTranslationsSourceSheet["FishModels"] = "Fish Models";
    SeriesTranslationsSourceSheet["Floors"] = "Floors";
    SeriesTranslationsSourceSheet["Fossils"] = "Fossils";
    SeriesTranslationsSourceSheet["Furniture"] = "Furniture";
    SeriesTranslationsSourceSheet["FurniturePatterns"] = "Furniture Patterns";
    SeriesTranslationsSourceSheet["FurnitureVariants"] = "Furniture Variants";
    SeriesTranslationsSourceSheet["HHAThemes"] = "HHA Themes";
    SeriesTranslationsSourceSheet["KKAlbums"] = "K.K. Albums";
    SeriesTranslationsSourceSheet["MarineSuit"] = "Marine Suit";
    SeriesTranslationsSourceSheet["Masks"] = "Masks";
    SeriesTranslationsSourceSheet["Pictures"] = "Pictures";
    SeriesTranslationsSourceSheet["Plants"] = "Plants";
    SeriesTranslationsSourceSheet["Posters"] = "Posters";
    SeriesTranslationsSourceSheet["Rugs"] = "Rugs";
    SeriesTranslationsSourceSheet["Shells"] = "Shells";
    SeriesTranslationsSourceSheet["Shoes"] = "Shoes";
    SeriesTranslationsSourceSheet["Socks"] = "Socks";
    SeriesTranslationsSourceSheet["Tools"] = "Tools";
    SeriesTranslationsSourceSheet["Tops"] = "Tops";
    SeriesTranslationsSourceSheet["Umbrella"] = "Umbrella";
    SeriesTranslationsSourceSheet["Walls"] = "Walls";
})(SeriesTranslationsSourceSheet = exports.SeriesTranslationsSourceSheet || (exports.SeriesTranslationsSourceSheet = {}));
var Version;
(function (Version) {
    Version["The100"] = "1.0.0";
    Version["The110"] = "1.1.0";
    Version["The120"] = "1.2.0";
    Version["The130"] = "1.3.0";
    Version["The140"] = "1.4.0";
    Version["The150"] = "1.5.0";
    Version["The160"] = "1.6.0";
    Version["The170"] = "1.7.0";
    Version["The180"] = "1.8.0";
    Version["The190"] = "1.9.0";
})(Version = exports.Version || (exports.Version = {}));
var RecipeSourceSheet;
(function (RecipeSourceSheet) {
    RecipeSourceSheet["Recipes"] = "Recipes";
})(RecipeSourceSheet = exports.RecipeSourceSheet || (exports.RecipeSourceSheet = {}));
var SecondaryShape;
(function (SecondaryShape) {
    SecondaryShape["H"] = "H";
    SecondaryShape["L"] = "L";
    SecondaryShape["N"] = "N";
})(SecondaryShape = exports.SecondaryShape || (exports.SecondaryShape = {}));
var Size;
(function (Size) {
    Size["The05X1"] = "0.5x1";
    Size["The15X15"] = "1.5x1.5";
    Size["The1X05"] = "1x0.5";
    Size["The1X1"] = "1x1";
    Size["The1X15"] = "1x1.5";
    Size["The1X2"] = "1x2";
    Size["The2X05"] = "2x0.5";
    Size["The2X1"] = "2x1";
    Size["The2X15"] = "2x1.5";
    Size["The2X2"] = "2x2";
    Size["The3X1"] = "3x1";
    Size["The3X2"] = "3x2";
    Size["The3X3"] = "3x3";
    Size["The4X3"] = "4x3";
    Size["The4X4"] = "4x4";
    Size["The5X5"] = "5x5";
})(Size = exports.Size || (exports.Size = {}));
var SizeCategory;
(function (SizeCategory) {
    SizeCategory["Large"] = "Large";
    SizeCategory["Medium"] = "Medium";
    SizeCategory["Small"] = "Small";
})(SizeCategory = exports.SizeCategory || (exports.SizeCategory = {}));
var SpeakerType;
(function (SpeakerType) {
    SpeakerType["Cheap"] = "Cheap";
    SpeakerType["HiFi"] = "Hi-fi";
    SpeakerType["Phono"] = "Phono";
    SpeakerType["Retro"] = "Retro";
})(SpeakerType = exports.SpeakerType || (exports.SpeakerType = {}));
var Style;
(function (Style) {
    Style["Active"] = "Active";
    Style["Cool"] = "Cool";
    Style["Cute"] = "Cute";
    Style["Elegant"] = "Elegant";
    Style["Gorgeous"] = "Gorgeous";
    Style["Simple"] = "Simple";
})(Style = exports.Style || (exports.Style = {}));
var Theme;
(function (Theme) {
    Theme["Comfy"] = "comfy";
    Theme["Everyday"] = "everyday";
    Theme["FairyTale"] = "fairy tale";
    Theme["Formal"] = "formal";
    Theme["Goth"] = "goth";
    Theme["Outdoorsy"] = "outdoorsy";
    Theme["Party"] = "party";
    Theme["Sporty"] = "sporty";
    Theme["Theatrical"] = "theatrical";
    Theme["Vacation"] = "vacation";
    Theme["Work"] = "work";
})(Theme = exports.Theme || (exports.Theme = {}));
var Chinese;
(function (Chinese) {
    Chinese["\u4F11\u95F2"] = "\u4F11\u95F2";
    Chinese["\u5DE5\u4F5C"] = "\u5DE5\u4F5C";
    Chinese["\u5EA6\u5047"] = "\u5EA6\u5047";
    Chinese["\u6050\u6016"] = "\u6050\u6016";
    Chinese["\u6237\u5916"] = "\u6237\u5916";
    Chinese["\u65E5\u5E38"] = "\u65E5\u5E38";
    Chinese["\u6B63\u5F0F"] = "\u6B63\u5F0F";
    Chinese["\u6D3E\u5BF9"] = "\u6D3E\u5BF9";
    Chinese["\u7AE5\u8BDD\u4E16\u754C"] = "\u7AE5\u8BDD\u4E16\u754C";
    Chinese["\u821E\u53F0"] = "\u821E\u53F0";
    Chinese["\u8FD0\u52A8"] = "\u8FD0\u52A8";
})(Chinese = exports.Chinese || (exports.Chinese = {}));
var ChineseTraditional;
(function (ChineseTraditional) {
    ChineseTraditional["\u5DE5\u4F5C"] = "\u5DE5\u4F5C";
    ChineseTraditional["\u5EA6\u5047"] = "\u5EA6\u5047";
    ChineseTraditional["\u6050\u6016"] = "\u6050\u6016";
    ChineseTraditional["\u60A0\u9592"] = "\u60A0\u9592";
    ChineseTraditional["\u6236\u5916\u6D3B\u52D5"] = "\u6236\u5916\u6D3B\u52D5";
    ChineseTraditional["\u65E5\u5E38\u751F\u6D3B"] = "\u65E5\u5E38\u751F\u6D3B";
    ChineseTraditional["\u6B63\u5F0F"] = "\u6B63\u5F0F";
    ChineseTraditional["\u6D3E\u5C0D"] = "\u6D3E\u5C0D";
    ChineseTraditional["\u7AE5\u8A71\u4E16\u754C"] = "\u7AE5\u8A71\u4E16\u754C";
    ChineseTraditional["\u821E\u53F0"] = "\u821E\u53F0";
    ChineseTraditional["\u904B\u52D5"] = "\u904B\u52D5";
})(ChineseTraditional = exports.ChineseTraditional || (exports.ChineseTraditional = {}));
var DutchEnum;
(function (DutchEnum) {
    DutchEnum["AlledaagsThema"] = "alledaags thema";
    DutchEnum["Feestthema"] = "feestthema";
    DutchEnum["FormeThema"] = "forme thema";
    DutchEnum["Griezelthema"] = "griezelthema";
    DutchEnum["Podiumthema"] = "podiumthema";
    DutchEnum["Sportthema"] = "sportthema";
    DutchEnum["Sprookjesthema"] = "sprookjesthema";
    DutchEnum["Vakantiethema"] = "vakantiethema";
    DutchEnum["Vrijetijdsthema"] = "vrijetijdsthema";
    DutchEnum["WeersbestendigThema"] = "weersbestendig thema";
    DutchEnum["Werkthema"] = "werkthema";
})(DutchEnum = exports.DutchEnum || (exports.DutchEnum = {}));
var French;
(function (French) {
    French["TenueDExt\u00E9rieur"] = "tenue d'ext\u00E9rieur";
    French["TenueDeD\u00E9tente"] = "tenue de d\u00E9tente";
    French["TenueDeF\u00EAte"] = "tenue de f\u00EAte";
    French["TenueDeSc\u00E8ne"] = "tenue de sc\u00E8ne";
    French["TenueDeTousLesJours"] = "tenue de tous les jours";
    French["TenueDeVacances"] = "tenue de vacances";
    French["TenueEffrayante"] = "tenue effrayante";
    French["TenueF\u00E9erique"] = "tenue f\u00E9erique";
    French["TenueF\u00E9\u00E9rique"] = "tenue f\u00E9\u00E9rique";
    French["TenueOfficielle"] = "tenue officielle";
    French["TenueProfessionnelle"] = "tenue professionnelle";
    French["TenueSport"] = "tenue sport";
})(French = exports.French || (exports.French = {}));
var German;
(function (German) {
    German["Alltagskleidung"] = "Alltagskleidung";
    German["Arbeitskleidung"] = "Arbeitskleidung";
    German["B\u00FChnenkleidung"] = "B\u00FChnenkleidung";
    German["Formellkleidung"] = "Formellkleidung";
    German["Freizeitkleidung"] = "Freizeitkleidung";
    German["Horrorkleidung"] = "Horrorkleidung";
    German["M\u00E4rchenkleidung"] = "M\u00E4rchenkleidung";
    German["Outdoorkleidung"] = "Outdoorkleidung";
    German["Partykleidung"] = "Partykleidung";
    German["Sportkleidung"] = "Sportkleidung";
    German["Urlaubskleidung"] = "Urlaubskleidung";
})(German = exports.German || (exports.German = {}));
var ID;
(function (ID) {
    ID["Daily"] = "Daily";
    ID["Fairyland"] = "Fairyland";
    ID["Fomal"] = "Fomal";
    ID["Horror"] = "Horror";
    ID["Outdoor"] = "Outdoor";
    ID["Party"] = "Party";
    ID["Relax"] = "Relax";
    ID["Sport"] = "Sport";
    ID["Stage"] = "Stage";
    ID["Vacation"] = "Vacation";
    ID["Work"] = "Work";
})(ID = exports.ID || (exports.ID = {}));
var Italian;
(function (Italian) {
    Italian["Confortevole"] = "confortevole";
    Italian["DaAriaAperta"] = "da aria aperta";
    Italian["DaFesta"] = "da festa";
    Italian["DaFiaba"] = "da fiaba";
    Italian["DaLavoro"] = "da lavoro";
    Italian["DaSport"] = "da sport";
    Italian["DaTeatro"] = "da teatro";
    Italian["DaVacanza"] = "da vacanza";
    Italian["Formale"] = "formale";
    Italian["Horror"] = "horror";
    Italian["PerTuttiIGiorni"] = "per tutti i giorni";
})(Italian = exports.Italian || (exports.Italian = {}));
var Japanese;
(function (Japanese) {
    Japanese["\u304A\u3057\u3054\u3068\u4ED5\u4E8B"] = "\u304A\u3057\u3054\u3068\u4ED5\u4E8B";
    Japanese["\u30A2\u30A6\u30C8\u30C9\u30A2"] = "\u30A2\u30A6\u30C8\u30C9\u30A2";
    Japanese["\u30B9\u30C6\u30FC\u30B8"] = "\u30B9\u30C6\u30FC\u30B8";
    Japanese["\u30B9\u30DD\u30FC\u30C4"] = "\u30B9\u30DD\u30FC\u30C4";
    Japanese["\u30C7\u30A4\u30EA\u30FC"] = "\u30C7\u30A4\u30EA\u30FC";
    Japanese["\u30D0\u30AB\u30F3\u30B9"] = "\u30D0\u30AB\u30F3\u30B9";
    Japanese["\u30D1\u30FC\u30C6\u30A3\u30FC"] = "\u30D1\u30FC\u30C6\u30A3\u30FC";
    Japanese["\u30D5\u30A9\u30FC\u30DE\u30EB"] = "\u30D5\u30A9\u30FC\u30DE\u30EB";
    Japanese["\u30DB\u30E9\u30FC"] = "\u30DB\u30E9\u30FC";
    Japanese["\u30E1\u30EB\u30D8\u30F3"] = "\u30E1\u30EB\u30D8\u30F3";
    Japanese["\u30EA\u30E9\u30C3\u30AF\u30B9"] = "\u30EA\u30E9\u30C3\u30AF\u30B9";
})(Japanese = exports.Japanese || (exports.Japanese = {}));
var Korean;
(function (Korean) {
    Korean["\uB370\uC77C\uB9AC"] = "\uB370\uC77C\uB9AC";
    Korean["\uB9B4\uB799\uC2A4"] = "\uB9B4\uB799\uC2A4";
    Korean["\uBA54\uB974\uD5E8"] = "\uBA54\uB974\uD5E8";
    Korean["\uBC14\uCE89\uC2A4"] = "\uBC14\uCE89\uC2A4";
    Korean["\uBE44\uC988\uB2C8\uC2A4"] = "\uBE44\uC988\uB2C8\uC2A4";
    Korean["\uC2A4\uD14C\uC774\uC9C0"] = "\uC2A4\uD14C\uC774\uC9C0";
    Korean["\uC2A4\uD3EC\uCE20"] = "\uC2A4\uD3EC\uCE20";
    Korean["\uC544\uC6C3\uB3C4\uC5B4"] = "\uC544\uC6C3\uB3C4\uC5B4";
    Korean["\uD30C\uD2F0"] = "\uD30C\uD2F0";
    Korean["\uD3EC\uBA40"] = "\uD3EC\uBA40";
    Korean["\uD638\uB7EC"] = "\uD638\uB7EC";
})(Korean = exports.Korean || (exports.Korean = {}));
var Russian;
(function (Russian) {
    Russian["\u0413\u043E\u0442\u0438\u0447\u0435\u0441\u043A\u0438\u0439"] = "\u0433\u043E\u0442\u0438\u0447\u0435\u0441\u043A\u0438\u0439";
    Russian["\u041E\u0442\u043F\u0443\u0441\u043A\u043D\u043E\u0439"] = "\u043E\u0442\u043F\u0443\u0441\u043A\u043D\u043E\u0439";
    Russian["\u041F\u043E\u0432\u0441\u0435\u0434\u043D\u0435\u0432\u043D\u044B\u0439"] = "\u043F\u043E\u0432\u0441\u0435\u0434\u043D\u0435\u0432\u043D\u044B\u0439";
    Russian["\u041F\u043E\u0445\u043E\u0434\u043D\u044B\u0439"] = "\u043F\u043E\u0445\u043E\u0434\u043D\u044B\u0439";
    Russian["\u041F\u0440\u0430\u0437\u0434\u043D\u0438\u0447\u043D\u044B\u0439"] = "\u043F\u0440\u0430\u0437\u0434\u043D\u0438\u0447\u043D\u044B\u0439";
    Russian["\u0420\u0430\u0431\u043E\u0447\u0438\u0439"] = "\u0440\u0430\u0431\u043E\u0447\u0438\u0439";
    Russian["\u0421\u043F\u043E\u0440\u0442\u0438\u0432\u043D\u044B\u0439"] = "\u0441\u043F\u043E\u0440\u0442\u0438\u0432\u043D\u044B\u0439";
    Russian["\u0423\u0434\u043E\u0431\u043D\u044B\u0439"] = "\u0443\u0434\u043E\u0431\u043D\u044B\u0439";
    Russian["\u0424\u0430\u043D\u0442\u0430\u0437\u0438\u0439\u043D\u044B\u0439"] = "\u0444\u0430\u043D\u0442\u0430\u0437\u0438\u0439\u043D\u044B\u0439";
    Russian["\u0424\u043E\u0440\u043C\u0430\u043B\u044C\u043D\u044B\u0439"] = "\u0444\u043E\u0440\u043C\u0430\u043B\u044C\u043D\u044B\u0439";
    Russian["\u042D\u0444\u0444\u0435\u043A\u0442\u043D\u044B\u0439"] = "\u044D\u0444\u0444\u0435\u043A\u0442\u043D\u044B\u0439";
})(Russian = exports.Russian || (exports.Russian = {}));
var ComfySourceSheet;
(function (ComfySourceSheet) {
    ComfySourceSheet["FashionThemes"] = "Fashion Themes";
})(ComfySourceSheet = exports.ComfySourceSheet || (exports.ComfySourceSheet = {}));
var Spanish;
(function (Spanish) {
    Spanish["Casual"] = "casual";
    Spanish["DeActividadesAlAireLibre"] = "de actividades al aire libre";
    Spanish["DeFantas\u00EDa"] = "de fantas\u00EDa";
    Spanish["DeFiesta"] = "de fiesta";
    Spanish["Deportivo"] = "deportivo";
    Spanish["Desenfadado"] = "desenfadado";
    Spanish["Formal"] = "formal";
    Spanish["G\u00F3tico"] = "g\u00F3tico";
    Spanish["Profesional"] = "profesional";
    Spanish["Teatral"] = "teatral";
    Spanish["Todoterreno"] = "todoterreno";
    Spanish["Vacacional"] = "vacacional";
})(Spanish = exports.Spanish || (exports.Spanish = {}));
var VariantID;
(function (VariantID) {
    VariantID["The0_0"] = "0_0";
    VariantID["The0_1"] = "0_1";
    VariantID["The0_2"] = "0_2";
    VariantID["The0_3"] = "0_3";
    VariantID["The0_4"] = "0_4";
    VariantID["The0_5"] = "0_5";
    VariantID["The0_6"] = "0_6";
    VariantID["The0_7"] = "0_7";
    VariantID["The1_0"] = "1_0";
    VariantID["The1_1"] = "1_1";
    VariantID["The1_2"] = "1_2";
    VariantID["The1_3"] = "1_3";
    VariantID["The1_4"] = "1_4";
    VariantID["The1_5"] = "1_5";
    VariantID["The1_6"] = "1_6";
    VariantID["The1_7"] = "1_7";
    VariantID["The2_0"] = "2_0";
    VariantID["The2_1"] = "2_1";
    VariantID["The2_2"] = "2_2";
    VariantID["The2_3"] = "2_3";
    VariantID["The2_4"] = "2_4";
    VariantID["The2_5"] = "2_5";
    VariantID["The2_6"] = "2_6";
    VariantID["The2_7"] = "2_7";
    VariantID["The3_0"] = "3_0";
    VariantID["The3_1"] = "3_1";
    VariantID["The3_2"] = "3_2";
    VariantID["The3_3"] = "3_3";
    VariantID["The3_4"] = "3_4";
    VariantID["The3_5"] = "3_5";
    VariantID["The3_6"] = "3_6";
    VariantID["The3_7"] = "3_7";
    VariantID["The4_0"] = "4_0";
    VariantID["The4_1"] = "4_1";
    VariantID["The4_2"] = "4_2";
    VariantID["The4_3"] = "4_3";
    VariantID["The4_4"] = "4_4";
    VariantID["The4_5"] = "4_5";
    VariantID["The4_6"] = "4_6";
    VariantID["The4_7"] = "4_7";
    VariantID["The5_0"] = "5_0";
    VariantID["The5_1"] = "5_1";
    VariantID["The5_2"] = "5_2";
    VariantID["The5_3"] = "5_3";
    VariantID["The5_4"] = "5_4";
    VariantID["The5_5"] = "5_5";
    VariantID["The5_6"] = "5_6";
    VariantID["The5_7"] = "5_7";
    VariantID["The6_0"] = "6_0";
    VariantID["The6_1"] = "6_1";
    VariantID["The6_2"] = "6_2";
    VariantID["The6_3"] = "6_3";
    VariantID["The6_4"] = "6_4";
    VariantID["The6_5"] = "6_5";
    VariantID["The6_6"] = "6_6";
    VariantID["The6_7"] = "6_7";
    VariantID["The7_0"] = "7_0";
    VariantID["The7_1"] = "7_1";
    VariantID["The7_2"] = "7_2";
    VariantID["The7_3"] = "7_3";
    VariantID["The7_4"] = "7_4";
    VariantID["The7_5"] = "7_5";
    VariantID["The7_6"] = "7_6";
    VariantID["The7_7"] = "7_7";
})(VariantID = exports.VariantID || (exports.VariantID = {}));
var KitType;
(function (KitType) {
    KitType["Normal"] = "Normal";
    KitType["Pumpkin"] = "Pumpkin";
    KitType["RainbowFeather"] = "Rainbow feather";
})(KitType = exports.KitType || (exports.KitType = {}));
var VariantTranslationsSourceSheet;
(function (VariantTranslationsSourceSheet) {
    VariantTranslationsSourceSheet["AccessoriesVariants"] = "Accessories Variants";
    VariantTranslationsSourceSheet["BagsVariants"] = "Bags Variants";
    VariantTranslationsSourceSheet["BottomsVariants"] = "Bottoms Variants";
    VariantTranslationsSourceSheet["CapsVariants"] = "Caps Variants";
    VariantTranslationsSourceSheet["DressesVariants"] = "Dresses Variants";
    VariantTranslationsSourceSheet["FurnitureVariants"] = "Furniture Variants";
    VariantTranslationsSourceSheet["MarineSuitVariants"] = "Marine Suit Variants";
    VariantTranslationsSourceSheet["MasksVariants"] = "Masks Variants";
    VariantTranslationsSourceSheet["ShoesVariants"] = "Shoes Variants";
    VariantTranslationsSourceSheet["SocksVariants"] = "Socks Variants";
    VariantTranslationsSourceSheet["TopsVariants"] = "Tops Variants";
})(VariantTranslationsSourceSheet = exports.VariantTranslationsSourceSheet || (exports.VariantTranslationsSourceSheet = {}));
var VfxType;
(function (VfxType) {
    VfxType["LightOff"] = "LightOff";
    VfxType["Random"] = "Random";
    VfxType["Synchro"] = "Synchro";
})(VfxType = exports.VfxType || (exports.VfxType = {}));
var WindowColor;
(function (WindowColor) {
    WindowColor["BlackMetal"] = "Black Metal";
    WindowColor["DarkWood"] = "Dark Wood";
    WindowColor["GreyMetal"] = "Grey Metal";
    WindowColor["GreyWood"] = "Grey Wood";
    WindowColor["LightWood"] = "Light Wood";
    WindowColor["NaturalWood"] = "Natural Wood";
    WindowColor["WhiteMetal"] = "White Metal";
    WindowColor["WhiteWood"] = "White Wood";
    WindowColor["Wood"] = "Wood";
})(WindowColor = exports.WindowColor || (exports.WindowColor = {}));
var WindowType;
(function (WindowType) {
    WindowType["Arch"] = "Arch";
    WindowType["Circle"] = "Circle";
    WindowType["FourPane"] = "Four Pane";
    WindowType["SinglePane"] = "Single Pane";
    WindowType["SlidingPane"] = "Sliding Pane";
})(WindowType = exports.WindowType || (exports.WindowType = {}));
