// animal-crossing/lib/types/Recipe.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.RecipeSourceSheet = exports.Version = exports.TranslationsSourceSheet = exports.ExchangeCurrency = exports.Category = exports.CardColor = void 0;
var CardColor;
(function (CardColor) {
    CardColor["Beige"] = "beige";
    CardColor["Blue"] = "blue";
    CardColor["Brick"] = "brick";
    CardColor["Brown"] = "brown";
    CardColor["Cream"] = "cream";
    CardColor["DarkGray"] = "dark gray";
    CardColor["Gold"] = "gold";
    CardColor["Green"] = "green";
    CardColor["LightGray"] = "light gray";
    CardColor["Orange"] = "orange";
    CardColor["Pink"] = "pink";
    CardColor["Purple"] = "purple";
    CardColor["Red"] = "red";
    CardColor["Silver"] = "silver";
    CardColor["White"] = "white";
    CardColor["Yellow"] = "yellow";
})(CardColor = exports.CardColor || (exports.CardColor = {}));
var Category;
(function (Category) {
    Category["Equipment"] = "Equipment";
    Category["Floors"] = "Floors";
    Category["Housewares"] = "Housewares";
    Category["Miscellaneous"] = "Miscellaneous";
    Category["Other"] = "Other";
    Category["Rugs"] = "Rugs";
    Category["Tools"] = "Tools";
    Category["WallMounted"] = "Wall-mounted";
    Category["Wallpaper"] = "Wallpaper";
})(Category = exports.Category || (exports.Category = {}));
var ExchangeCurrency;
(function (ExchangeCurrency) {
    ExchangeCurrency["NookMiles"] = "Nook Miles";
})(ExchangeCurrency = exports.ExchangeCurrency || (exports.ExchangeCurrency = {}));
var TranslationsSourceSheet;
(function (TranslationsSourceSheet) {
    TranslationsSourceSheet["Accessories"] = "Accessories";
    TranslationsSourceSheet["Bags"] = "Bags";
    TranslationsSourceSheet["Bottoms"] = "Bottoms";
    TranslationsSourceSheet["Caps"] = "Caps";
    TranslationsSourceSheet["Craft"] = "Craft";
    TranslationsSourceSheet["Doorplates"] = "Doorplates";
    TranslationsSourceSheet["Dresses"] = "Dresses";
    TranslationsSourceSheet["Etc"] = "ETC";
    TranslationsSourceSheet["EventItems"] = "Event Items";
    TranslationsSourceSheet["Fence"] = "Fence";
    TranslationsSourceSheet["Floors"] = "Floors";
    TranslationsSourceSheet["Fossils"] = "Fossils";
    TranslationsSourceSheet["Furniture"] = "Furniture";
    TranslationsSourceSheet["Masks"] = "Masks";
    TranslationsSourceSheet["Plants"] = "Plants";
    TranslationsSourceSheet["Rugs"] = "Rugs";
    TranslationsSourceSheet["Shells"] = "Shells";
    TranslationsSourceSheet["Shoes"] = "Shoes";
    TranslationsSourceSheet["Tools"] = "Tools";
    TranslationsSourceSheet["Tops"] = "Tops";
    TranslationsSourceSheet["Umbrella"] = "Umbrella";
    TranslationsSourceSheet["Walls"] = "Walls";
})(TranslationsSourceSheet = exports.TranslationsSourceSheet || (exports.TranslationsSourceSheet = {}));
var Version;
(function (Version) {
    Version["The100"] = "1.0.0";
    Version["The110"] = "1.1.0";
    Version["The120"] = "1.2.0";
    Version["The130"] = "1.3.0";
    Version["The140"] = "1.4.0";
    Version["The150"] = "1.5.0";
    Version["The160"] = "1.6.0";
    Version["The170"] = "1.7.0";
    Version["The180"] = "1.8.0";
})(Version = exports.Version || (exports.Version = {}));
var RecipeSourceSheet;
(function (RecipeSourceSheet) {
    RecipeSourceSheet["Recipes"] = "Recipes";
})(RecipeSourceSheet = exports.RecipeSourceSheet || (exports.RecipeSourceSheet = {}));
