// animal-crossing/lib/types/SeasonsAndEvents.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.YearEnum = exports.UnlockMethodEnum = exports.Type = exports.Version = exports.TranslationsSourceSheet = exports.SeasonsAndEventSourceSheet = exports.Availability = void 0;
var Availability;
(function (Availability) {
    Availability["EveryYear"] = "Every year";
    Availability["SpecificYears"] = "Specific years";
    Availability["SpecificYearsNHOnly"] = "Specific years (NH only)";
    Availability["SpecificYearsSHOnly"] = "Specific years (SH only)";
})(Availability = exports.Availability || (exports.Availability = {}));
var SeasonsAndEventSourceSheet;
(function (SeasonsAndEventSourceSheet) {
    SeasonsAndEventSourceSheet["SeasonsAndEvents"] = "Seasons and Events";
})(SeasonsAndEventSourceSheet = exports.SeasonsAndEventSourceSheet || (exports.SeasonsAndEventSourceSheet = {}));
var TranslationsSourceSheet;
(function (TranslationsSourceSheet) {
    TranslationsSourceSheet["Constellations"] = "Constellations";
    TranslationsSourceSheet["Craft"] = "Craft";
    TranslationsSourceSheet["Events"] = "Events";
    TranslationsSourceSheet["HHAThemes"] = "HHA Themes";
    TranslationsSourceSheet["Shells"] = "Shells";
})(TranslationsSourceSheet = exports.TranslationsSourceSheet || (exports.TranslationsSourceSheet = {}));
var Version;
(function (Version) {
    Version["The100"] = "1.0.0";
    Version["The110"] = "1.1.0";
    Version["The120"] = "1.2.0";
    Version["The130"] = "1.3.0";
    Version["The140"] = "1.4.0";
    Version["The150"] = "1.5.0";
    Version["The160"] = "1.6.0";
    Version["The170"] = "1.7.0";
    Version["The180"] = "1.8.0";
    Version["The190"] = "1.9.0";
})(Version = exports.Version || (exports.Version = {}));
var Type;
(function (Type) {
    Type["BasegameEvent"] = "Basegame event";
    Type["CraftingSeason"] = "Crafting season";
    Type["NookShoppingEvent"] = "Nook Shopping event";
    Type["ShoppingSeason"] = "Shopping season";
    Type["SpecialEvent"] = "Special event";
    Type["ZodiacSeason"] = "Zodiac season";
})(Type = exports.Type || (exports.Type = {}));
var UnlockMethodEnum;
(function (UnlockMethodEnum) {
    UnlockMethodEnum["BCATA"] = "BCAT (a)";
    UnlockMethodEnum["BCATB"] = "BCAT (b)";
    UnlockMethodEnum["BCATC"] = "BCAT (c)";
    UnlockMethodEnum["NTP"] = "NTP";
})(UnlockMethodEnum = exports.UnlockMethodEnum || (exports.UnlockMethodEnum = {}));
var YearEnum;
(function (YearEnum) {
    YearEnum["Any"] = "Any";
})(YearEnum = exports.YearEnum || (exports.YearEnum = {}));
